spec/imsc-hrm.html

```diff
@@ -471,6 +471,14 @@ <h2>Architecture</h2>
       "Intermediate Synchronic Document">Intermediate Synchronic Documents</a>, allowing glyphs to be copied into the Presentation
       Buffer instead of rendered, a more costly operation.</p>
 
+      <p class="note">The HRM permits a maximum rate of 12 <a>Intermediate Synchronic Documents</a> per second. This is
+      ultimately limited by the BDraw parameter and is intended to capture processing and presentation overhead. When
+      converting a [[CEA-608]] signal to IMSC, it is therefore impossible to create a separate <a>IMSC Document
+      Instance</a> for every [[CEA-608]] packet, which are sampled at approximately 59.94 Hz. It is instead preferable
+      to coalesce sequences of [[CEA-608]] packets into complete lines and paragraphs before creating a <a>IMSC Document
```

Comment:
The leap to creating Document Instances seems a bit too far. Also, it is surely possible to use other groupings of 608 packets to reduce the overall frequency.

```suggestion
      converting a [[CEA-608]] signal to IMSC, it is therefore impossible to create one or more <a>IMSC Document
      Instances</a> that generate an <a>Intermediate Synchronic Documents</a> for every [[CEA-608]] packet, which are sampled at approximately 59.94 Hz. It is instead preferable
      to coalesce sequences of [[CEA-608]] packets into longer groupings, such as words, phrases, complete lines or paragraphs before creating an <a>IMSC Document
```